### PR TITLE
EZP-30417: Add notifications to the password reset view

### PR DESCRIPTION
**Pull request by czechmarcin**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-30417](https://jira.ez.no/browse/EZP-30417)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Added notifications to the password reset view in order to show validation errors.

#### Checklist:
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.notifications.js [hunk 1]
```diff
@@ -6,10 +6,10 @@
         const { onShow, label, message, rawPlaceholdersMap = {} } = detail;
         // @TODO: Unify 'error' and 'danger' label in 3.0.
         const configKey = label === 'danger' ? 'error' : label;
-        const config = eZ.adminUiConfig.notifications[configKey];
+        const config = eZ.adminUiConfig ? eZ.adminUiConfig.notifications[configKey] : null;
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.notifications.js]: I don't think this is a good approach. This timeout for notifications should be respected.

File: src/bundle/Resources/public/js/scripts/admin.notifications.js [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
         const timeout = config ? config.timeout : 0;
         const container = doc.createElement('div');
-        let finalMessage = eZ.helpers.text.escapeHTML(message);
+        let finalMessage = eZ.helpers.text ? eZ.helpers.text.escapeHTML(message) : message;
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.notifications.js]: This was added for security reasons. Don't think we should ever bypass this.

File: src/bundle/Resources/public/js/scripts/admin.notifications.js (hunk 1 above)
Comment (czechmarcin) [reply src/bundle/Resources/public/js/scripts/admin.notifications.js]: AdminUI is not the only place when we need this. The Password Reset view is in `Security` scope. Should I implement the `adminUiConfig` in `src/bundle/Resources/views/Security/base.html.twig` instead?

Comment (dew326) [reply src/bundle/Resources/public/js/scripts/admin.notifications.js]: Good question. My concern is that those notifications will not respect the timeouts. Maybe we should introduce some limited adminUiConfig? Only with needed things?
@Nattfarinn @webhdx

File: src/bundle/Resources/public/js/scripts/admin.notifications.js (hunk 2 above)
Comment (czechmarcin) [reply src/bundle/Resources/public/js/scripts/admin.notifications.js]: In order to have any helpers in `Security/base.html.twig` I should implement the [addConfig](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/views/layout.html.twig#L20) function. Would it be okay if I separate this code into other `twig` file and include it in both files `Security/base.html.twig` and `layout.html.twig`?

**Comment by czechmarcin** [comment]
Problem solved another way, please refer to PRs:
https://github.com/ezsystems/repository-forms/pull/288
https://github.com/ezsystems/ezplatform-user/pull/30

**Pull request closed without merging**